asdf/tests/test_extension.py

```diff
@@ -110,6 +117,14 @@ def select_tag(self, obj, tags, ctx):
         return "select_tag result"
 
 
+class MinimalCompressor(Compressor):
```

Comment:
Haha

Reply:
`def compress(data): return b''` may be lossy, but it has a great compression ratio!